Documentation/General/productPlanning/planning.tex

```diff
@@ -126,36 +125,29 @@ \section{User stories of features}
 When I click on the 'mute' button,\\
 Then the music should be muted.
 
-As a user,\\
-When I join a room,\\
-Then I should see the latest messages sent in this room.
-
-\section{User stories of defects (if applicable)}
-As of the time of writing, no defects have been discovered.
-
-\section{User stories of technical improvements (if applicable)}
-As of the time of writing, no technical improvements were made.
-
 \section{User stories of know-how acquisition}
+As a developer, \\
+When I do not know something, \\
+Then I will search on the internet for relevant information.
 
 \section{Initial release plan}
-
+%3.5. Initial release plan (milestones, MRFs per release)
+Our milestones consists only of the sprint number .
+For example, if a feature shall be made in Sprint 3 it has milestone Sprint 3. 
```

Comment:
'Each sprint has a corresponding milestone. For example, if a feature will be developed during sprint 3, the corresponding GitHub issue will have milestone 3. This makes sure that the roadmap outlines are closely followed and progress can be easily determined by using the GitHub tools.'